brolit_lite.sh

```
@@ -2310,53 +2310,50 @@ function show_backup_information() {
         # Mount storage box
         mount_storage_box "${storage_box_directory}"
 
+        # Temporary file to store each parallel process output
+        temp_file=$(mktemp)
+
         # Loop through project directories in the mounted storage box
         for project_directory in $(ls --color=never "${storage_box_directory}/${BACKUP_BORG_GROUP}/${HOSTNAME}/projects-online/site"); do
```

Shellcheck

[warning] Iterating over ls output is fragile. Use globs.

(SC2045)